po/CREDITS.json

```diff
@@ -39,4 +39,8 @@
 		"lang": "Swedish",
 		"url": "https://github.com/vanillajonathan"
 	}
+	"Quentin PAGÈS": {
+		"lang": "Occitan",
+		"url": "https://github.com/Mejans"
+	},	
```

Comment:
please move "," from line 45 to 41 so the file will be correct

Reply:
Done (sorry I entered the review mode that I know nothing about)

Reply:
> Done (sorry I entered the review mode that I know nothing about)

No worries, thank you for your work 😊